tc/sgx/trusted_worker_manager/enclave/iohandler_enclave.cpp

```diff
@@ -22,23 +22,26 @@
 #include "iohandler_enclave.h"
 #include "enclave_utils.h"
 
-/*
-  Dummy ecall (non root function) to support independent
-  interface for iohandler
-*/
+
+/**
+ * Dummy ecall (non root function) to support an independent
+ * interface for the iohandler.
+ */
 void ecall_dummy() {}
 
-/*
-  Executes given io command by invoking ocall and stores result of
-  io execution in result buffer
-  handlerId - identifier for iohandler
-  command - IO command that needs to be executed outside enclave
-  result - status of IO operation
-  resultSize - Maximum size of the result buffer
-  inBuf - buffer having input data
-  inBufSize - max size of buffer having input data
-  outBuf - placeholder for output data
-  outBufSize - max size of buffer having output data
+
+/**
+ * Executes the given I/O command by invoking ocall and stores the result of
+ * I/O execution in result buffer.
+ *
+ * @param handlerId  Identifier for iohandler
+ * @param command    IO command that needs to be executed outside enclave
```

Comment:
@param commandSize  size of the IO command

Reply:
Fixed